lib/scss/inc/_variables.scss

```diff
@@ -18,10 +18,9 @@ $text-muted-color: #606060;
 $black: #000;
 $white: #fff;
 
-
 // Body
 $body-bg: $white;
-$body-color: $black;
+$body-color: #707070;
```

Comment:
I think we decided earlier to keep the body color black to make the documentation more readable. However, if there has been another discussion to change it back, we can apply this change.

Ping @deekoder